pkg/tbtc/node.go

```diff
@@ -53,6 +258,8 @@ func newNode(
 	}
 }
 
+// TODO: Extract the DKG-specific code into a separate file `pkg/tbtc/dkg.go`
```

Comment:
Let's put this TODO just after the TODO about unit tests for `node.go`.

Reply:
Done in 083ec8d0cd8bbf586163024e2073eb64c15c2d43.